python/processing/__init__.py

```diff
@@ -29,3 +29,99 @@
 from .algfactory import ProcessingAlgFactory
 
 alg = ProcessingAlgFactory()
+
+
+# "Forward declare" functions which will be patched in when the Processing plugin loads:
```

Comment:
would it be worth adding a note somewhere that this needs to be keep up synchronized?

could we use an`@overload` in general.py so the docstrings are written only once?

Reply:
Ok, addressed